joschahenningsen/TUM-Live
# feat: add location to "Scheduled" lectures on course page

### Motivation and Context
Under scheduled lectures a new link to lecture hall webpage on NavigaTUM is added.
The issue: https://github.com/TUM-Dev/gocast/issues/1193

### Description
The lecture hall information of scheduled lectures is used to query NavigaTUM.

### Steps for Testing
Prerequisites:
- 1 Module

1. Log in
2. Create a new scheduled lecture for the module
3. Go to lecture homepage on TUMLive

Local copies of TUMLive don't update the lecture hall information, so no link is displayed there. However a reviewer can manually set a RoomCode for the lecture in order to observe the change.
